modules/isp-setup.nix

```diff
@@ -0,0 +1,39 @@
+{ pkgs, config, lib, ... }:
+
+let
+  inherit (lib) types mkEnableOption mkOption;
+
+  cfg = config.services.isp-setup;
+
+  copyISPfiles = pkgs.writeShellScriptBin "copy-isp-files" ''
+    if [[ ! -f "/var/nvidia/nvcam" ]]; then
+      ${lib.getExe' pkgs.coreutils "mkdir"} -p /var/nvidia/nvcam
+      ${lib.getExe' pkgs.coreutils "cp"} -r ${cfg.path}/. /var/nvidia/nvcam
+      ${lib.getExe' pkgs.coreutils "chmod"} -R 644 /var/nvidia/nvcam
+    fi
+  '';
+in
+{
+  options = {
+    services.isp-setup = {
+      
+      enable = mkEnableOption "ISP setup";
+
+      path = mkOption {
```

Comment:
This should probably be an option under `hardware.jetpack-nixos` instead of something separate.
Could the option just be a list of packages which provide isp files? It'd be empty by default, so I don't see the need for an `enable` field.